program/src/instructions/close_margin_account.rs

```diff
@@ -30,18 +27,8 @@ pub struct CloseMarginAccount<'info> {
     pub system_program: Program<'info, System>,
 }
 
-impl<'info> Validate<'info> for CloseMarginAccount<'info> {
-    fn validate(&self) -> Result<()> {
-        if self.margin_account.pools_attached > 0 {
```

Comment:
Intentionally remove since we no longer keep this in sync. 

Also this doesn't really matter (closing a margin account if there are pools) since an empty margin account is equivalent, and that can happen regardless of how many pools there are attached.